run-test.cmd

```diff
@@ -8,7 +8,7 @@ set LOCATION=%1
 set RUNTIME_PATH=%2
 
 if "%LOCATION%" == "" set LOCATION=%~dp0\bin\Windows_NT.AnyCPU.Debug
-if "%RUNTIME_PATH%" == "" set RUNTIME_PATH=%~dp0\Tools\testdotnetcli
+if "%RUNTIME_PATH%" == "" set RUNTIME_PATH=%~dp0\bin\runtime\netcoreapp-Windows_NT-Debug-x64
```

Comment:
Hm, I thought the old path still had "shared\9.9.9" underneath it, but maybe I'm misremembering... I think this should be the sub-folder, actually.

But yeah... can we just delete this? I bet if anyone is using this stuff, it's probably the Unix `.sh` version anyways.